src/H5Fvfd_swmr.c

```diff
@@ -883,6 +943,18 @@ H5F_vfd_swmr_writer_end_of_tick(H5F_t *f, hbool_t wait_for_reader)
         HGOTO_ERROR(H5E_FILE, H5E_CANTSET, FAIL, "unable to insert entry into the EOT queue")
 
 done:
+    /* Kent: Calcuate the processing time and write the time info to the log file */
+    if (shared->vfd_swmr_log_on == true) {
+        if (H5_timer_get_times(shared->vfd_swmr_log_start_time, &current_time) < 0)
+            HGOTO_ERROR(H5E_FILE, H5E_CANTGET, FAIL, "can't get time from H5_timer_get_times")
+        end_elapsed_time = current_time.elapsed;
+        log_msg          = HDmalloc(48);
```

Comment:
Error check for HDmalloc(48)?
Use define for 48 instead of a constant?

Reply:
Vailin, I can define a constant. I didn't do this because this part of code is only valid when the logging is turned on. Also this is an example. I am not sure if we want to keep this.

Reply:
ok